doc/config_rsmtool.rst

```diff
@@ -84,7 +84,9 @@ The name for the column containing the human scores in the training data. If set
 
 .. note::
 
-    All responses with non-numeric values in either ``train_label_column`` or ``test_label_column`` and/or those with non-numeric values for relevant features will be automatically excluded from model training and evaluation. By default, zero scores in either ``train_label_column`` or ``test_label_column`` will also be excluded. See :ref:`exclude_zero_scores_rsmtool` if you want to keep responses with zero scores.
+    1. All responses with non-numeric values in either ``train_label_column`` or ``test_label_column`` and/or those with non-numeric values for relevant features will be automatically excluded from model training and evaluation. By default, zero scores in either ``train_label_column`` or ``test_label_column`` will also be excluded. See :ref:`exclude_zero_scores_rsmtool` if you want to keep responses with zero scores.
+
+    2. If ``train_label_column`` and ``test_label_column`` contain floating point scores, they may be rounded for some of the evaluation methods, e.g. when computing kappa and the confusion matrices. 
```

Comment:
"May be rounded for some" is too vague in my opinion. If you do have floating point scores, you want to know what is going on. "They will be rounded for the following evaluations: ....".
Are we sure we are not rounding them for correlations too?

Reply:
Can you make the change you think is appropriate?

Reply:
Re-open the issue and remove it from this PR? I think it's more then just wording - it's about checking where we round the scores in the code too.

Reply:
The issue is still open. I will edit the PR description.